### DISCUSSION TITLE
Question: With tabs for multiple files, are you supposed to be able to click on the tabs to change files?

### DISCUSSION THREAD

**Discussion by octokatherine** [code-hike/codehike#85] [Q&A] [answered]
I was looking at the Tabs demo on [https://codehike.org/demo/filenames](https://codehike.org/demo/filenames) and expected it to change tabs when I clicked on `styles.css` but it didn't

**Comment by pomber** [comment, marked as answer]
Yes, it should change the tab, but it's not implemented yet. 

Gonna fix it today. Thanks for the reminder!

**Reply by pomber** [reply]
Fixed in #86. It should work now. Thanks again!
